research/conditional/moe_layers/cont_moe_designs/send_result_only_to_top1_token.py

```diff
@@ -1,11 +1,27 @@
+import torch
 from lizrd.core import misc
-from lizrd.support import ash
-from research.conditional.archive.rogue_code import set_highest_index_one
 from research.conditional.moe_layers.continuous_moe import ContinuousMoeBaseClass
 from research.conditional.utils.misc_tools import stable_softmax_temperature
 
 
-@ash.check("... dinp -> ... dout")
+def set_highest_index_one(tensor: torch.Tensor) -> torch.Tensor:
```

Comment:
is it used anywhere or spill from other PR?

Reply:
Apparently class below was using this function from archive

Reply:
So I moved it here and removed archive file